src/PhpDoc/PhpDocNodeResolver.php

```diff
@@ -212,6 +217,45 @@ private function resolveThrowsTags(PhpDocNode $phpDocNode, NameScope $nameScope)
 		return new ThrowsTag(TypeCombinator::union(...$types));
 	}
 
+	private function resolveDeprecatedTag(PhpDocNode $phpDocNode, NameScope $nameScope): ?\PHPStan\PhpDoc\Tag\DeprecatedTag
+	{
+		foreach ($phpDocNode->getDeprecatedTagValues() as $deprecatedTagValue) {
```

Comment:
🤔 @ondrejmirtes so this whole block of code "reassembles" multiple line deprecation comments. Should that go _here_ or in the phpdoc-parser library? It didn't occur to be previously. But now that we have the code in phpdoc-parser it popped up in my head.

Reply:
Try making it part of `phpdoc-parser` so that the node description field already has the text.

Reply:
Thanks, I will do that